src/core/mount.c

```diff
@@ -653,6 +650,11 @@ static int mount_add_extras(Mount *m) {
                         return r;
         }
 
+        if (mount_is_credentials(m)) {
+                u->ignore_on_isolate = true;
```

Comment:
Ah, we set `ignore_on_isolate = true` in `mount_init`, i.e. for all mount units. So this is unneeded.